cedar-policy/src/api.rs

```diff
@@ -1507,6 +1509,7 @@ impl Template {
     /// create the EST from the policy text or CST instead, as the conversion
     /// to AST is lossy. ESTs generated by this method will reflect the AST and
     /// not the original policy syntax.
+    #[allow(unused)] //Feature: partial-eval
```

Comment:
This could be `#[cfg_attr(feature = "partial-eval", allow(unused))]`

Likewise elsewhere